iphone/cli/commands/_buildModule.js

```diff
@@ -558,15 +628,17 @@ iOSModuleBuilder.prototype.createUniversalBinary = function createUniversalBinar
 		args.push('-headers');
 		args.push(headerPath);
 	}
-	lib = findLib('macosx');
-	if (lib instanceof Error) {
-		this.logger.warn(__('The module is missing 64-bit support of macos. Ignoring mac target for this module...'));
-	} else {
-		args.push(buildType);
-		args.push(lib);
-		if (!this.isFramework) {
-			args.push('-headers');
-			args.push(headerPath);
+	if (this.isMacOSEnabled) {
+		lib = findLib('macosx');
+		if (lib instanceof Error) {
+			this.logger.warn(__('The module is missing 64-bit support of macos. Ignoring mac target for this module...'));
```

Comment:
The warning is misleading. It's missing macOS support entirely, not 64-bit support.

Reply:
Also, they've "turned on" Mac support, so shouldn't this be an Error, not a warning log and move on?